plugins/database/mysql/connection_producer.go

```diff
@@ -29,6 +37,8 @@ type mySQLConnectionProducer struct {
 	MaxConnectionLifetimeRaw interface{} `json:"max_connection_lifetime" mapstructure:"max_connection_lifetime" structs:"max_connection_lifetime"`
 	Username                 string      `json:"username" mapstructure:"username" structs:"username"`
 	Password                 string      `json:"password" mapstructure:"password" structs:"password"`
+	AuthType                 string      `json:"auth_type" mapstructure:"auth_type" structs:"auth_type"`
+	Credentials              string      `json:"credentials" mapstructure:"credentials" structs:"credentials"`
```

Comment:
Is `Credentials` being using anywhere?

Reply:
Sorry, ignore this comment. I had made some changes to this branch locally and lost the reference to it.